### PR TITLE
Replace jQuery with JS in service.md code sample

### PR DESCRIPTION
**Pull request by Techn1x**
https://github.com/ember-learn/guides-source/issues/195

### CODE REVIEW

File: guides/v3.4.0/tutorial/service.md [hunk 1]
```diff
@@ -197,7 +197,11 @@ export default Component.extend({
   didInsertElement() {
     this._super(...arguments);
     let mapElement = this.maps.getMapElement(this.location);
-    this.$('.map-container').append(mapElement);
+    
+    let componentElement = document.getElementById(this.elementId);
```
Comment (locks) [inline guides/v3.4.0/tutorial/service.md]: I think you can do `this.element` instead and it will have the same effect.

Comment (Techn1x) [reply guides/v3.4.0/tutorial/service.md]: Ah! Learning something new every day :) Fixed

**Comment by Techn1x** [comment]
Improved again (using `querySelector()` instead of `getElementsByClassName()`)

**Comment by Techn1x** [comment]
Sorry, I effed up this branch/PR when trying to sync with the 3.5 files. New PR is here https://github.com/ember-learn/guides-source/pull/207

**Pull request closed without merging**